example/src/main/tut/typeclass/Ord.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: docs
+title:  "Ord"
+---
+
+# Ord
+
+*Ord is used for totally ordered datatypes.*
```

Comment:
I wonder if we should (eventually) have a `PartialOrder` typeclass, or even a `PreOrder`.